### ISSUE TITLE
Chinese Localization

### ISSUE THREAD

**Issue by GamMa0892** [Werebearguy/AssortedCrazyThings#20]
Can't post link in steam workshop, so I'm trying here lol

This is a simple chinese localization.please feel free to add it.

https://mega.nz/file/Cd1kkY4D#dGFyuGvtCVg3vTg67Zdib7IL52pPo53k8BRlSFnKOPw

It hasn't been checked too much, but it should be fine.

**Comment by direwolf420** [comment]
Hi, thanks for taking the time to translate the mod, really appreciated. I have consulted a chinese-speaking modder to proofread and this is his and my feedback for formatting and wording

### Formatting/Syntax
* As per guidelines, text that starts with a special character like `{` requires the full text to be surrounded in (english) quotation marks `" "`. Due to language differences, an english text that does not have them might need them for the chinese text, like here: `BigEnemyDisappear: The {0} disappeared... for now` -> `BigEnemyDisappear: {0}消失了...暂时` it's missing the `" "` around it (`BigEnemyDisappear: "{0}消失了...暂时"`)

* Instances of `...` should be changed to `……`, for example in `Tier3: 我不确定它是否应该继续吃...` or the above example

* instances of the `' '` should be changed to `“ ”`, for example in `EnableSmartCursor: 启用'智能光标'自动放置火把`

* In general, no spaces (` `). These should have their spaces removed: `AntiqueChestTile.MapEntry_Locked`, `ConcatenateTwo`

* Plurlization pattern: `{^0:minute;minutes}` is indicating a plurlization pattern which changes the word based on the number given. In chinese, this is not needed 95% of the time, so just change it to `分钟`. Do the same with the other plurlizations, they all start wtih `{^`

* Text that is originally capitalized `LIKE THIS` should be surrounded in `【】`

### Wording
* `CuteSlimeQueenBuff Description` is missing the "A" `一`

* (this is taken word for word from the other person, since I am not quite sure what it means) there is some inconsistency in the translation, specifically adding a 'number word' at start of description, some of them, which the thing the pet is referring to, can have multiple instances in a world, which I would rather put a number word there, while the translation has not. the words are like "一个", "两只". (Referring to `CuteSlimeQueenBuff`)

* `HintClientConfig` has a "H"

* `PlaceablesFunctional`, `PlaceablesDecorative` and `Hint` have untranslated text

* `SlimeBeaconTile` "Background" refers to the sky background where slime is falling from the sky, this is not the correct wording in the translation

This is all, if you have questions ask here or if you use discord add me `@putan`

**Comment by ush0892** [comment]
Sorry for the late reply ><

This is a new file and should have fixed these problem.

https://mega.nz/file/vF9yBR4C#6ycm5PlrXqlhu0gYz3QlGBG_OXWhq_MOBOP81V7gOrY

**Comment by direwolf420** [comment]
Thanks, most of the things are proper now, some stuff still needs work:

untranslated:
`Gitgud -> TimesDied` (and you put quotes there when the shouldn't be)
`Campfire.WyvernCampfireNearby`

commented:
`// ShortName: 手套` should not have the `//`

pluralization:
there's still instances of `{^` that are not translated/removed

Other:
HarvesterRelicItem and HarvesterRelicColoredItem need to change to this:
```
HarvesterRelicItem: {
    DisplayName: "{$NPCs.HarvesterBoss.DisplayName}圣物"
    Tooltip: ""
}

HarvesterRelicColoredItem: {
    DisplayName: "{$Items.HarvesterRelicItem.DisplayName}(彩色)"
    Tooltip: ""
}
```

You added tooltips to PetAccessory stuff, english one does not have them, please revert

**Comment by ush0892** [comment]
ok, this is the third version.

https://mega.nz/file/XRNhlCyJ#vq7unr-ZSeiN-I9-skxzxDdIdW_MMie_t2-l3bYQNf0

I really like your humor on the wiki, so I added those tooltips. have deleted it, sorry.

**Comment by direwolf420** [comment]
Thanks, what do you like to be called in the credits? ush0892 or GamMa?

**Comment by ush0892** [comment]
GamMa, thank you. ^^

**Comment by direwolf420** [comment]
If you plan on updating the localization in the future, please concider creating a [Pull Request](https://github.com/tModLoader/tModLoader/wiki/Contributing-Localization#github-3) instead, it does not require uploading files to mega, and allows me to review the file easier

**Comment by GamMa0892** [comment]
I don't use github often, will try next time. ( ・ д・)ゞ

**Comment by direwolf420** [comment]
Your localization will be included in the next mod update soon (v1.4.5). Thanks again
